### PR TITLE
chore: add background toggle for examples

### LINKED ISSUE #300: Improve Docs: Add dark vs light mode for code examples
### Description

Follow design seen here:
https://sap.github.io/fundamental-ngx/docs/action-bar

image

CORUI-6166

### PR DESCRIPTION
**Pull request by jbadan**
### Description
Adds light and dark mode for examples as seen in Fundamental NGX and Fiori Fundamentals.

fixes #300

### CODE REVIEW

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 1]
```diff
@@ -1,30 +1,59 @@
...
+        return (
+            <div className={outerDivClasses}>
+                <Toggle
+                    className='frDocs-tile__background-toggle'
+                    inputProps={{'aria-label': 'Toggle background color'}}
+                    onChange={this.handleToggle}
+                    size='xs' />
+                {centered
+                    ? (
+                        <div className='fd-tile__content'>
```
Comment (bcullman) [inline src/_playground/documentation/DocsTile/DocsTile.js]: can you move centerStyle to `fd-tile__content`?

Comment (jbadan) [reply src/_playground/documentation/DocsTile/DocsTile.js]: Sure, I wasn't sure if this was by design. See the DocsText component in the same file. All styles are stored here as well.

Comment (bcullman) [reply src/_playground/documentation/DocsTile/DocsTile.js]: feel free chose then Do here, or open a separate PR to further cleanse CSS

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 2]
```diff
@@ -1,30 +1,49 @@
...
-    );
-};
+        return (
+            <div className={outerDivClasses}>
+                <Toggle
+                    className='frDocs-tile__background-toggle'
+                    inputProps={{'aria-label': 'Toggle background color'}}
+                    onChange={this.handleToggle}
+                    size='xs' />
+                <div className='fd-tile__content'>
```
Comment (bcullman) [inline src/_playground/documentation/DocsTile/DocsTile.js]: `centered` can be added here, rather than adding another div container.

**Comment by bcullman** [comment]
@jbadan: can you add a giphy here?

File: src/_playground/App.scss [hunk 3]
```diff
@@ -90,7 +90,15 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
...
-                background-color: '#f3f4f5'
+                
+                .frDocs-tile__background-toggle {
+                    display: flex;
+                    justify-content: flex-end;
+                    margin: 15px;
+                }
+            }
+            .frDocs-Content__tile-background {
+                background: #edeef0;
```
[the remark this answers is not in the document]
Comment (bcullman) [reply src/_playground/App.scss]: looks a bit abrupt. Consider a transition `transition: background-color .25s;`

this is what fundamentals uses.

File: src/_playground/documentation/DocsTile/DocsTile.js [hunk 4]
```diff
@@ -1,30 +1,56 @@
...
+            }
+        );
+
+        return (
+            <div className={outerDivClasses}>
+                <Toggle
+                    className='frDocs-tile__background-toggle'
+                    inputProps={{'aria-label': 'Toggle background color'}}
+                    onChange={this.handleToggle}
+                    size='xs' />
```
Comment (greg-a-smith) [inline src/_playground/documentation/DocsTile/DocsTile.js]: I think we should add "Toggle background" as the `children` for `Toggle` so it's clear _what_ this toggle is for?  And now looking at the docs site, maybe add `children` as a prop explaining it's the label. (Or maybe #322 already is doing this?)

Comment (bcullman) [reply src/_playground/documentation/DocsTile/DocsTile.js]: perhaps add text too (right now it is toggle background - in the future it should be dark mode):

image

Comment (jbadan) [reply src/_playground/documentation/DocsTile/DocsTile.js]: I think https://github.com/SAP/fundamental-react/issues/321 should handle the label confusion as well

**Pull request merged**